tests/pyspark-notebook/test_spark.py

```diff
@@ -16,3 +16,20 @@ def test_spark_shell(container: TrackedContainer) -> None:
     )
 
     assert "res0: Int = 2" in logs, "spark-shell does not work"
+
+
+def test_pandas_version(container: TrackedContainer) -> None:
+    """Checking if pandas is installed and its version is 1.5.3
+    This is only needed for Spark before 4.0 see #1924"""
+    logs = container.run_and_wait(
+        timeout=60,
+        tty=True,
+        command=[
+            "start.sh",
+            "python",
+            "-c",
+            "import pandas; print(pandas.__version__)",
+        ],
+    )
+
+    assert "1.5.3" in logs, "pandas version is not 1.5.3"
```

Comment:
Instead of all this code, just add a file [here](https://github.com/jupyter/docker-stacks/tree/main/tests/pyspark-notebook/units) with the following content:

```
# Copyright (c) Jupyter Development Team.
# Distributed under the terms of the Modified BSD License.

import pandas
assert pandas.__version__ == "1.5.3"
```

Reply:
This is much easier and works well.